src/authentication/oauth2Auth.ts

```diff
@@ -300,17 +300,13 @@ export class Oauth2Auth extends AlfrescoApiClient {
     redirectLogin(): void {
         if (this.config.oauth2.implicitFlow && typeof window !== 'undefined') {
             let href = this.composeImplicitLoginUrl();
-            const afterLoginUriSegment = this.isRedirectionUrl() ? window.location.hash.split('&')[0] : '';
-            if (afterLoginUriSegment && afterLoginUriSegment !== '/') {
-                this.storage.setItem('loginFragment', afterLoginUriSegment.replace('#', '').trim());
-            }
             window.location.href = href;
             this.emit('implicit_redirect', href);
         }
     }
 
     isRedirectionUrl() {
-        return window.location.hash  && window.location.hash.indexOf('&') !== -1 && window.location.hash.split('&')[0].indexOf('session_state') === -1;
```

Comment:
are we sure we want to remove this index check ?

Reply:
yes, it was pointlessly added to check if extra parameters were there, in the end and url request with no parameters won't be saved and will create a problem